### PR TITLE
cli flow command for migrating project planning branches from cplace to cplace-project-planning repository.

### PR DESCRIPTION
**Pull request by shariquehaq**
cli flow command for migrating project planning branches from cplace to cplace-project-planning repository.

### CODE REVIEW

File: src/commands/flow/ProjectPlanningRefactor.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import * as path from 'path';
+import {PROJECT_PLANNING_PARAM} from './models';
+import {Repository} from '../../git';
+
+export class ProjectPlanningRefactor implements ICommand {
+    private static readonly PATH_TO_PROJECT_PLANNING_REPO: string = 'pathProjectPlanning';
+    private readonly CPLACE_TEMP_REPO: string = 'cplace-temp-repo';
+
+    private targetBranchName: string;
+    private cplaceRepo: simpleGit.Git;
```
Comment (rose-m) [inline src/commands/flow/ProjectPlanningRefactor.ts]: Type for `cplaceRepo` and `projectPlanningRepo` is wrong

File: src/commands/flow/models.ts [hunk 2]
```diff
@@ -7,4 +7,12 @@ import {ReleaseNumber} from './ReleaseNumber';
 export interface IBranchDetails extends IGitBranchDetails {
     customer: string;
     version: ReleaseNumber;
-}
\ No newline at end of file
+}
+
+export const PROJECT_PLANNING_PARAM = '-- cf.cplace.projektplanung cf.cplace.rplanRxfImport cf.cplace.integration.ganttAndRiskmanagementAndTaskBoard' +
```
Comment (rose-m) [inline src/commands/flow/models.ts]: This constant should not be in the general `models.ts` file but inside the corresponding command

File: src/commands/flow/ProjectPlanningRefactor.ts (hunk 1 above)
Comment (shariquehaq) [reply src/commands/flow/ProjectPlanningRefactor.ts]: 👍

File: src/commands/flow/models.ts (hunk 2 above)
Comment (shariquehaq) [reply src/commands/flow/models.ts]: 👍

**Comment by shariquehaq** [comment]
> LGTM 👍 Great work :) I haven't tested myself - did anyone else test it so far, too? :)

Well, i used this to refactor project plannning 2 times. Also @csemrau and me tested different use cases together before big refactor. Looks good to us. But no body tested it apart from me.

**Pull request merged**